lib/router.js

```diff
@@ -68,13 +68,15 @@ export default class Router {
     this.window.history[this.historyMethod] = this.origChangeStateMethod;
   }
 
-  navigate(fragment, stateUpdate = {}) {
+  navigate(fragment, stateUpdate = {}, {dontUpdateUrl = false} = {}) {
```

Comment:
I considered naming the flag `updateUrl`, but slightly prefer false by default param, so went with `dontUpdateUrl`